[rospy.client][INFO] 2020-01-28 17:14:39,187: init_node, name[/ros_test], pid[10991]
[xmlrpc][INFO] 2020-01-28 17:14:39,188: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:14:39,188: Started XML-RPC server [http://huai:36311/]
[rospy.init][INFO] 2020-01-28 17:14:39,188: ROS Slave URI: [http://huai:36311/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:14:39,188: _ready: http://huai:36311/
[rospy.registration][INFO] 2020-01-28 17:14:39,189: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 17:14:39,189: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 17:14:39,288: registered with master
[rospy.rosout][INFO] 2020-01-28 17:14:39,288: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:14:39,290: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:14:39,291: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:14:39,519: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-01-28 17:14:41,538: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 17:14:41,540: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2020-01-28 17:14:41,540: signal-2
[rospy.core][INFO] 2020-01-28 17:14:41,702: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 17:15:14,332: init_node, name[/ros_test], pid[12214]
[xmlrpc][INFO] 2020-01-28 17:15:14,339: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:15:14,339: Started XML-RPC server [http://huai:35989/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:15:14,340: _ready: http://huai:35989/
[rospy.init][INFO] 2020-01-28 17:15:14,340: ROS Slave URI: [http://huai:35989/]
[rospy.registration][INFO] 2020-01-28 17:15:14,345: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 17:15:14,346: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 17:15:14,441: registered with master
[rospy.rosout][INFO] 2020-01-28 17:15:14,441: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:15:14,443: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:15:14,444: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:15:14,583: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 17:15:16,988: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 17:15:26,927: topic[/cam0/image_raw] adding connection to [http://huai:35871/], count 0
[rospy.internal][WARNING] 2020-01-28 17:17:01,419: Unknown error initiating TCP/IP socket to huai:45395 (http://huai:35871/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 17:17:01,420: topic[/cam0/image_raw] removing connection to http://huai:35871/
[rospy.core][INFO] 2020-01-28 17:23:20,288: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 17:23:20,290: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 17:23:20,290: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 17:23:20,290: signal-2
[rospy.core][INFO] 2020-01-28 17:23:20,455: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 17:26:27,296: init_node, name[/ros_test], pid[25986]
[xmlrpc][INFO] 2020-01-28 17:26:27,305: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:26:27,305: Started XML-RPC server [http://huai:45625/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:26:27,305: _ready: http://huai:45625/
[rospy.init][INFO] 2020-01-28 17:26:27,306: ROS Slave URI: [http://huai:45625/]
[xmlrpc][INFO] 2020-01-28 17:26:27,310: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-01-28 17:26:27,310: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-01-28 17:26:27,407: registered with master
[rospy.rosout][INFO] 2020-01-28 17:26:27,407: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:26:27,409: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:26:27,410: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:26:27,681: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 17:26:29,886: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 17:26:37,341: topic[/cam0/image_raw] adding connection to [http://huai:36627/], count 0
[rospy.internal][WARNING] 2020-01-28 17:27:30,668: Unknown error initiating TCP/IP socket to huai:34491 (http://huai:36627/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 17:27:30,668: topic[/cam0/image_raw] removing connection to http://huai:36627/
[rospy.core][INFO] 2020-01-28 17:41:34,536: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 17:41:34,538: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 17:41:34,538: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 17:41:34,538: signal-2
[rospy.core][INFO] 2020-01-28 17:41:34,641: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 17:41:58,066: init_node, name[/ros_test], pid[8718]
[xmlrpc][INFO] 2020-01-28 17:41:58,070: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:41:58,071: Started XML-RPC server [http://huai:33277/]
[rospy.init][INFO] 2020-01-28 17:41:58,071: ROS Slave URI: [http://huai:33277/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:41:58,071: _ready: http://huai:33277/
[rospy.registration][INFO] 2020-01-28 17:41:58,087: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 17:41:58,087: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 17:41:58,172: registered with master
[rospy.rosout][INFO] 2020-01-28 17:41:58,172: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:41:58,173: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:41:58,174: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:41:58,400: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 17:42:00,721: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 17:42:07,268: topic[/cam0/image_raw] adding connection to [http://huai:43151/], count 0
[rospy.internal][WARNING] 2020-01-28 17:43:35,030: Unknown error initiating TCP/IP socket to huai:34131 (http://huai:43151/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 17:43:35,030: topic[/cam0/image_raw] removing connection to http://huai:43151/
[rospy.core][INFO] 2020-01-28 17:43:38,412: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 17:43:38,414: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 17:43:38,414: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 17:43:38,414: signal-2
[rospy.core][INFO] 2020-01-28 17:43:38,579: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 17:49:49,449: init_node, name[/ros_test], pid[18966]
[xmlrpc][INFO] 2020-01-28 17:49:49,454: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:49:49,455: Started XML-RPC server [http://huai:37763/]
[rospy.init][INFO] 2020-01-28 17:49:49,455: ROS Slave URI: [http://huai:37763/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:49:49,455: _ready: http://huai:37763/
[xmlrpc][INFO] 2020-01-28 17:49:49,466: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-01-28 17:49:49,466: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-01-28 17:49:49,556: registered with master
[rospy.rosout][INFO] 2020-01-28 17:49:49,556: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:49:49,557: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:49:49,558: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:49:49,767: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 17:49:51,971: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 17:50:02,598: topic[/cam0/image_raw] adding connection to [http://huai:46069/], count 0
[rospy.internal][WARNING] 2020-01-28 17:51:37,105: Unknown error initiating TCP/IP socket to huai:57983 (http://huai:46069/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 17:51:37,105: topic[/cam0/image_raw] removing connection to http://huai:46069/
[rospy.core][INFO] 2020-01-28 17:55:34,510: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 17:55:34,512: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 17:55:34,512: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 17:55:34,512: signal-2
[rospy.core][INFO] 2020-01-28 17:55:34,629: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 17:56:59,882: init_node, name[/ros_test], pid[29677]
[xmlrpc][INFO] 2020-01-28 17:56:59,887: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 17:56:59,887: Started XML-RPC server [http://huai:41395/]
[rospy.init][INFO] 2020-01-28 17:56:59,887: ROS Slave URI: [http://huai:41395/]
[rospy.impl.masterslave][INFO] 2020-01-28 17:56:59,888: _ready: http://huai:41395/
[rospy.registration][INFO] 2020-01-28 17:56:59,899: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 17:56:59,899: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 17:56:59,988: registered with master
[rospy.rosout][INFO] 2020-01-28 17:56:59,988: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 17:56:59,990: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 17:56:59,994: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 17:57:00,240: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 17:57:02,444: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 17:57:10,932: topic[/cam0/image_raw] adding connection to [http://huai:45511/], count 0
[rospy.internal][WARNING] 2020-01-28 17:58:45,411: Unknown error initiating TCP/IP socket to huai:42281 (http://huai:45511/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 17:58:45,412: topic[/cam0/image_raw] removing connection to http://huai:45511/
[rospy.core][INFO] 2020-01-28 18:01:07,093: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:01:07,096: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:01:07,096: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:01:07,096: signal-2
[rospy.core][INFO] 2020-01-28 18:01:07,229: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 18:01:28,174: init_node, name[/ros_test], pid[6471]
[xmlrpc][INFO] 2020-01-28 18:01:28,179: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 18:01:28,179: Started XML-RPC server [http://huai:39795/]
[rospy.init][INFO] 2020-01-28 18:01:28,179: ROS Slave URI: [http://huai:39795/]
[rospy.impl.masterslave][INFO] 2020-01-28 18:01:28,180: _ready: http://huai:39795/
[xmlrpc][INFO] 2020-01-28 18:01:28,184: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-01-28 18:01:28,184: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-01-28 18:01:28,280: registered with master
[rospy.rosout][INFO] 2020-01-28 18:01:28,280: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 18:01:28,282: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 18:01:28,282: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 18:01:28,506: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 18:01:30,745: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 18:01:37,166: topic[/cam0/image_raw] adding connection to [http://huai:46405/], count 0
[rospy.internal][WARNING] 2020-01-28 18:03:11,646: Unknown error initiating TCP/IP socket to huai:41527 (http://huai:46405/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 18:03:11,646: topic[/cam0/image_raw] removing connection to http://huai:46405/
[rospy.core][INFO] 2020-01-28 18:10:15,117: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:10:15,119: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:10:15,119: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:10:15,119: signal-2
[rospy.core][INFO] 2020-01-28 18:10:15,285: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 18:15:53,345: init_node, name[/ros_test], pid[21553]
[xmlrpc][INFO] 2020-01-28 18:15:53,350: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 18:15:53,350: Started XML-RPC server [http://huai:33187/]
[rospy.init][INFO] 2020-01-28 18:15:53,351: ROS Slave URI: [http://huai:33187/]
[rospy.impl.masterslave][INFO] 2020-01-28 18:15:53,351: _ready: http://huai:33187/
[rospy.registration][INFO] 2020-01-28 18:15:53,356: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 18:15:53,356: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 18:15:53,451: registered with master
[rospy.rosout][INFO] 2020-01-28 18:15:53,451: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 18:15:53,453: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 18:15:53,454: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 18:15:53,566: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 18:15:55,975: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 18:16:00,792: topic[/cam0/image_raw] adding connection to [http://huai:34917/], count 0
[rospy.internal][WARNING] 2020-01-28 18:17:35,313: Unknown error initiating TCP/IP socket to huai:51941 (http://huai:34917/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 18:17:35,313: topic[/cam0/image_raw] removing connection to http://huai:34917/
[rospy.core][INFO] 2020-01-28 18:18:02,260: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:18:02,261: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:18:02,262: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:18:02,262: signal-2
[rospy.core][INFO] 2020-01-28 18:18:02,427: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 18:18:33,498: init_node, name[/ros_test], pid[28858]
[xmlrpc][INFO] 2020-01-28 18:18:33,499: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2020-01-28 18:18:33,500: ROS Slave URI: [http://huai:42503/]
[xmlrpc][INFO] 2020-01-28 18:18:33,500: Started XML-RPC server [http://huai:42503/]
[rospy.impl.masterslave][INFO] 2020-01-28 18:18:33,501: _ready: http://huai:42503/
[rospy.registration][INFO] 2020-01-28 18:18:33,510: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 18:18:33,510: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 18:18:33,600: registered with master
[rospy.rosout][INFO] 2020-01-28 18:18:33,601: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 18:18:33,602: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 18:18:33,603: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 18:18:33,705: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 18:18:36,110: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 18:18:42,428: topic[/cam0/image_raw] adding connection to [http://huai:38447/], count 0
[rospy.internal][WARNING] 2020-01-28 18:20:16,946: Unknown error initiating TCP/IP socket to huai:37813 (http://huai:38447/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 18:20:16,946: topic[/cam0/image_raw] removing connection to http://huai:38447/
[rospy.core][INFO] 2020-01-28 18:27:14,768: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:27:14,770: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:27:14,771: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:27:14,771: signal-2
[rospy.core][INFO] 2020-01-28 18:27:14,888: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 18:43:17,782: init_node, name[/ros_test], pid[19768]
[xmlrpc][INFO] 2020-01-28 18:43:17,782: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 18:43:17,782: Started XML-RPC server [http://huai:45741/]
[rospy.impl.masterslave][INFO] 2020-01-28 18:43:17,782: _ready: http://huai:45741/
[rospy.registration][INFO] 2020-01-28 18:43:17,783: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 18:43:17,783: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 18:43:17,783: ROS Slave URI: [http://huai:45741/]
[rospy.init][INFO] 2020-01-28 18:43:17,783: registered with master
[rospy.rosout][INFO] 2020-01-28 18:43:17,783: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 18:43:17,785: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 18:43:17,786: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 18:43:18,034: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 18:43:20,239: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 18:43:30,064: topic[/cam0/image_raw] adding connection to [http://huai:34515/], count 0
[rospy.internal][WARNING] 2020-01-28 18:45:04,574: Unknown error initiating TCP/IP socket to huai:57351 (http://huai:34515/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 18:45:04,574: topic[/cam0/image_raw] removing connection to http://huai:34515/
[rospy.core][INFO] 2020-01-28 18:45:52,521: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:45:52,523: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:45:52,523: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:45:52,523: signal-2
[rospy.core][INFO] 2020-01-28 18:45:52,657: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 18:46:17,430: init_node, name[/ros_test], pid[27612]
[xmlrpc][INFO] 2020-01-28 18:46:17,430: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 18:46:17,430: Started XML-RPC server [http://huai:42069/]
[rospy.init][INFO] 2020-01-28 18:46:17,430: ROS Slave URI: [http://huai:42069/]
[rospy.impl.masterslave][INFO] 2020-01-28 18:46:17,431: _ready: http://huai:42069/
[rospy.registration][INFO] 2020-01-28 18:46:17,431: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 18:46:17,431: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 18:46:17,531: registered with master
[rospy.rosout][INFO] 2020-01-28 18:46:17,531: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 18:46:17,533: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 18:46:17,533: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 18:46:17,789: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 18:46:19,993: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 18:46:24,561: topic[/cam0/image_raw] adding connection to [http://huai:38913/], count 0
[rospy.internal][WARNING] 2020-01-28 18:47:59,048: Unknown error initiating TCP/IP socket to huai:42095 (http://huai:38913/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 18:47:59,048: topic[/cam0/image_raw] removing connection to http://huai:38913/
[rospy.core][INFO] 2020-01-28 18:51:17,779: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 18:51:17,782: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 18:51:17,782: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 18:51:17,782: signal-2
[rospy.core][INFO] 2020-01-28 18:51:18,398: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 20:51:36,001: init_node, name[/ros_test], pid[18066]
[xmlrpc][INFO] 2020-01-28 20:51:36,005: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 20:51:36,006: Started XML-RPC server [http://huai:39263/]
[rospy.init][INFO] 2020-01-28 20:51:36,006: ROS Slave URI: [http://huai:39263/]
[rospy.impl.masterslave][INFO] 2020-01-28 20:51:36,006: _ready: http://huai:39263/
[rospy.registration][INFO] 2020-01-28 20:51:36,018: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 20:51:36,018: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 20:51:36,107: registered with master
[rospy.rosout][INFO] 2020-01-28 20:51:36,107: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 20:51:36,108: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 20:51:36,109: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 20:51:36,291: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 20:51:38,595: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 20:52:01,340: topic[/cam0/image_raw] adding connection to [http://huai:33085/], count 0
[rospy.internal][WARNING] 2020-01-28 20:53:35,950: Unknown error initiating TCP/IP socket to huai:37201 (http://huai:33085/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 20:53:35,950: topic[/cam0/image_raw] removing connection to http://huai:33085/
[rospy.core][INFO] 2020-01-28 21:11:46,722: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 21:11:46,724: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 21:11:46,724: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 21:11:46,725: signal-2
[rospy.core][INFO] 2020-01-28 21:11:46,889: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 21:23:34,977: init_node, name[/ros_test], pid[13805]
[xmlrpc][INFO] 2020-01-28 21:23:34,981: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 21:23:34,982: Started XML-RPC server [http://huai:32951/]
[rospy.init][INFO] 2020-01-28 21:23:34,982: ROS Slave URI: [http://huai:32951/]
[rospy.impl.masterslave][INFO] 2020-01-28 21:23:34,982: _ready: http://huai:32951/
[rospy.registration][INFO] 2020-01-28 21:23:34,987: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 21:23:34,988: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 21:23:35,083: registered with master
[rospy.rosout][INFO] 2020-01-28 21:23:35,083: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 21:23:35,084: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 21:23:35,085: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 21:23:35,354: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 21:23:37,561: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 21:23:49,520: topic[/cam0/image_raw] adding connection to [http://huai:42765/], count 0
[rospy.core][INFO] 2020-01-28 21:33:21,533: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 21:33:21,535: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 21:33:21,535: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-28 21:33:21,535: topic[/cam0/image_raw] removing connection to http://huai:42765/
[rospy.internal][WARNING] 2020-01-28 21:33:21,536: Unknown error initiating TCP/IP socket to huai:42767 (http://huai:42765/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-01-28 21:33:21,538: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 841, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-01-28 21:33:21,538: signal-2
[rospy.core][INFO] 2020-01-28 21:33:21,646: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 21:36:37,340: init_node, name[/ros_test], pid[24608]
[xmlrpc][INFO] 2020-01-28 21:36:37,346: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 21:36:37,347: Started XML-RPC server [http://huai:41209/]
[rospy.init][INFO] 2020-01-28 21:36:37,347: ROS Slave URI: [http://huai:41209/]
[rospy.impl.masterslave][INFO] 2020-01-28 21:36:37,348: _ready: http://huai:41209/
[rospy.registration][INFO] 2020-01-28 21:36:37,360: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 21:36:37,360: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 21:36:37,448: registered with master
[rospy.rosout][INFO] 2020-01-28 21:36:37,448: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 21:36:37,450: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 21:36:37,450: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 21:36:37,701: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 21:36:39,727: topic[/cam0/image_raw] adding connection to [http://huai:42765/], count 0
[rospy.internal][INFO] 2020-01-28 21:36:39,945: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][WARNING] 2020-01-28 21:36:52,838: Unknown error initiating TCP/IP socket to huai:42767 (http://huai:42765/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 21:36:52,838: topic[/cam0/image_raw] removing connection to http://huai:42765/
[rospy.internal][INFO] 2020-01-28 21:36:56,196: topic[/cam0/image_raw] adding connection to [http://huai:35677/], count 0
[rospy.core][INFO] 2020-01-28 21:39:37,113: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 21:39:37,115: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 21:39:37,115: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-28 21:39:37,115: topic[/cam0/image_raw] removing connection to http://huai:35677/
[rospy.impl.masterslave][INFO] 2020-01-28 21:39:37,115: signal-2
[rospy.core][INFO] 2020-01-28 21:39:37,218: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 21:39:43,714: init_node, name[/ros_test], pid[27496]
[xmlrpc][INFO] 2020-01-28 21:39:43,722: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 21:39:43,722: Started XML-RPC server [http://huai:43379/]
[rospy.init][INFO] 2020-01-28 21:39:43,722: ROS Slave URI: [http://huai:43379/]
[rospy.impl.masterslave][INFO] 2020-01-28 21:39:43,722: _ready: http://huai:43379/
[rospy.registration][INFO] 2020-01-28 21:39:43,723: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 21:39:43,723: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 21:39:43,822: registered with master
[rospy.rosout][INFO] 2020-01-28 21:39:43,823: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 21:39:43,825: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 21:39:43,825: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 21:39:44,064: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 21:39:46,292: topic[/cam0/image_raw] adding connection to [http://huai:35677/], count 0
[rospy.internal][INFO] 2020-01-28 21:39:46,473: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][WARNING] 2020-01-28 21:39:50,001: Unknown error initiating TCP/IP socket to huai:57445 (http://huai:35677/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 21:39:50,001: topic[/cam0/image_raw] removing connection to http://huai:35677/
[rospy.internal][INFO] 2020-01-28 21:39:52,657: topic[/cam0/image_raw] adding connection to [http://huai:43573/], count 0
[rospy.internal][WARNING] 2020-01-28 21:41:27,162: Unknown error initiating TCP/IP socket to huai:47429 (http://huai:43573/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 21:41:27,162: topic[/cam0/image_raw] removing connection to http://huai:43573/
[rospy.core][INFO] 2020-01-28 21:43:47,288: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 21:43:47,290: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 21:43:47,290: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 21:43:47,290: signal-2
[rospy.core][INFO] 2020-01-28 21:43:47,423: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 21:45:06,590: init_node, name[/ros_test], pid[4741]
[xmlrpc][INFO] 2020-01-28 21:45:06,590: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 21:45:06,590: Started XML-RPC server [http://huai:33605/]
[rospy.init][INFO] 2020-01-28 21:45:06,590: ROS Slave URI: [http://huai:33605/]
[rospy.impl.masterslave][INFO] 2020-01-28 21:45:06,590: _ready: http://huai:33605/
[rospy.registration][INFO] 2020-01-28 21:45:06,591: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 21:45:06,591: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 21:45:06,691: registered with master
[rospy.rosout][INFO] 2020-01-28 21:45:06,691: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 21:45:06,693: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 21:45:06,693: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 21:45:06,902: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 21:45:09,307: topic[/cam0/image_raw] adding connection to [http://huai:43375/], count 0
[rospy.internal][INFO] 2020-01-28 21:45:09,502: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.core][INFO] 2020-01-28 21:45:53,495: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 21:45:53,501: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 21:45:53,501: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-28 21:45:53,501: topic[/cam0/image_raw] removing connection to http://huai:43375/
[rospy.impl.masterslave][INFO] 2020-01-28 21:45:53,502: signal-2
[rospy.core][INFO] 2020-01-28 21:45:53,619: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 21:46:09,649: init_node, name[/ros_test], pid[9485]
[xmlrpc][INFO] 2020-01-28 21:46:09,659: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 21:46:09,666: Started XML-RPC server [http://huai:42095/]
[rospy.init][INFO] 2020-01-28 21:46:09,667: ROS Slave URI: [http://huai:42095/]
[rospy.impl.masterslave][INFO] 2020-01-28 21:46:09,667: _ready: http://huai:42095/
[rospy.registration][INFO] 2020-01-28 21:46:09,672: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 21:46:09,672: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 21:46:09,767: registered with master
[rospy.rosout][INFO] 2020-01-28 21:46:09,767: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 21:46:09,769: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 21:46:09,770: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 21:46:10,017: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 21:46:12,325: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 21:46:18,556: topic[/cam0/image_raw] adding connection to [http://huai:40275/], count 0
[rospy.internal][WARNING] 2020-01-28 21:47:53,066: Unknown error initiating TCP/IP socket to huai:51711 (http://huai:40275/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 21:47:53,066: topic[/cam0/image_raw] removing connection to http://huai:40275/
[rospy.core][INFO] 2020-01-28 22:03:08,908: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 22:03:08,910: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 22:03:08,910: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 22:03:08,910: signal-2
[rospy.core][INFO] 2020-01-28 22:03:09,076: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 22:04:05,546: init_node, name[/ros_test], pid[27685]
[xmlrpc][INFO] 2020-01-28 22:04:05,555: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 22:04:05,556: Started XML-RPC server [http://huai:36491/]
[rospy.init][INFO] 2020-01-28 22:04:05,556: ROS Slave URI: [http://huai:36491/]
[rospy.impl.masterslave][INFO] 2020-01-28 22:04:05,556: _ready: http://huai:36491/
[rospy.registration][INFO] 2020-01-28 22:04:05,567: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 22:04:05,567: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 22:04:05,657: registered with master
[rospy.rosout][INFO] 2020-01-28 22:04:05,657: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 22:04:05,658: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 22:04:05,659: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 22:04:05,881: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 22:04:08,084: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 22:04:24,344: topic[/cam0/image_raw] adding connection to [http://huai:44339/], count 0
[rospy.internal][WARNING] 2020-01-28 22:05:58,835: Unknown error initiating TCP/IP socket to huai:45029 (http://huai:44339/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 22:05:58,835: topic[/cam0/image_raw] removing connection to http://huai:44339/
[rospy.core][INFO] 2020-01-28 22:08:22,277: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 22:08:22,279: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 22:08:22,279: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 22:08:22,279: signal-2
[rospy.core][INFO] 2020-01-28 22:08:22,412: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 22:08:44,949: init_node, name[/ros_test], pid[4283]
[xmlrpc][INFO] 2020-01-28 22:08:44,954: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 22:08:44,955: Started XML-RPC server [http://huai:37665/]
[rospy.init][INFO] 2020-01-28 22:08:44,955: ROS Slave URI: [http://huai:37665/]
[rospy.impl.masterslave][INFO] 2020-01-28 22:08:44,955: _ready: http://huai:37665/
[rospy.registration][INFO] 2020-01-28 22:08:44,960: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 22:08:44,961: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 22:08:45,056: registered with master
[rospy.rosout][INFO] 2020-01-28 22:08:45,056: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 22:08:45,058: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 22:08:45,058: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 22:08:45,194: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 22:08:47,600: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 22:08:52,842: topic[/cam0/image_raw] adding connection to [http://huai:37883/], count 0
[rospy.internal][WARNING] 2020-01-28 22:10:27,328: Unknown error initiating TCP/IP socket to huai:45021 (http://huai:37883/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 22:10:27,328: topic[/cam0/image_raw] removing connection to http://huai:37883/
[rospy.core][INFO] 2020-01-28 22:15:58,727: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 22:15:58,729: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 22:15:58,729: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 22:15:58,729: signal-2
[rospy.core][INFO] 2020-01-28 22:15:58,846: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 22:43:47,441: init_node, name[/ros_test], pid[1014]
[xmlrpc][INFO] 2020-01-28 22:43:47,463: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 22:43:47,463: Started XML-RPC server [http://huai:35723/]
[rospy.init][INFO] 2020-01-28 22:43:47,463: ROS Slave URI: [http://huai:35723/]
[rospy.impl.masterslave][INFO] 2020-01-28 22:43:47,465: _ready: http://huai:35723/
[xmlrpc][INFO] 2020-01-28 22:43:47,466: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-01-28 22:43:47,466: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-01-28 22:43:47,566: registered with master
[rospy.rosout][INFO] 2020-01-28 22:43:47,566: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 22:43:47,568: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 22:43:47,569: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 22:43:47,825: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 22:43:50,128: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 22:44:36,908: topic[/cam0/image_raw] adding connection to [http://huai:33663/], count 0
[rospy.internal][WARNING] 2020-01-28 22:46:11,322: Unknown error initiating TCP/IP socket to huai:40305 (http://huai:33663/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 22:46:11,322: topic[/cam0/image_raw] removing connection to http://huai:33663/
[rospy.core][INFO] 2020-01-28 22:53:40,890: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 22:53:40,892: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 22:53:40,892: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 22:53:40,893: signal-2
[rospy.core][INFO] 2020-01-28 22:53:41,057: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 22:59:22,318: init_node, name[/ros_test], pid[17336]
[xmlrpc][INFO] 2020-01-28 22:59:22,324: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 22:59:22,325: Started XML-RPC server [http://huai:43725/]
[rospy.init][INFO] 2020-01-28 22:59:22,325: ROS Slave URI: [http://huai:43725/]
[rospy.impl.masterslave][INFO] 2020-01-28 22:59:22,326: _ready: http://huai:43725/
[rospy.registration][INFO] 2020-01-28 22:59:22,330: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 22:59:22,330: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 22:59:22,426: registered with master
[rospy.rosout][INFO] 2020-01-28 22:59:22,426: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 22:59:22,428: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 22:59:22,428: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 22:59:22,681: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 22:59:24,784: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 22:59:58,368: topic[/cam0/image_raw] adding connection to [http://huai:45009/], count 0
[rospy.internal][WARNING] 2020-01-28 23:02:25,756: Unknown error initiating TCP/IP socket to huai:49939 (http://huai:45009/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-28 23:02:25,756: topic[/cam0/image_raw] removing connection to http://huai:45009/
[rospy.core][INFO] 2020-01-28 23:03:45,541: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 23:03:45,543: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 23:03:45,543: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-28 23:03:45,544: signal-2
[rospy.core][INFO] 2020-01-28 23:03:45,660: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 23:04:25,947: init_node, name[/ros_test], pid[29166]
[xmlrpc][INFO] 2020-01-28 23:04:25,948: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-28 23:04:25,949: Started XML-RPC server [http://huai:42177/]
[rospy.init][INFO] 2020-01-28 23:04:25,949: ROS Slave URI: [http://huai:42177/]
[rospy.impl.masterslave][INFO] 2020-01-28 23:04:25,949: _ready: http://huai:42177/
[rospy.registration][INFO] 2020-01-28 23:04:25,950: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 23:04:25,950: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 23:04:26,049: registered with master
[rospy.rosout][INFO] 2020-01-28 23:04:26,050: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 23:04:26,051: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 23:04:26,052: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 23:04:26,257: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 23:04:28,563: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 23:04:31,580: topic[/cam0/image_raw] adding connection to [http://huai:38383/], count 0
[rospy.core][INFO] 2020-01-28 23:04:58,531: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 23:04:58,533: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 23:04:58,533: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-28 23:04:58,533: topic[/cam0/image_raw] removing connection to http://huai:38383/
[rospy.impl.masterslave][INFO] 2020-01-28 23:04:58,533: signal-2
[rospy.core][INFO] 2020-01-28 23:04:58,714: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-28 23:05:07,931: init_node, name[/ros_test], pid[30958]
[xmlrpc][INFO] 2020-01-28 23:05:07,935: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2020-01-28 23:05:07,936: ROS Slave URI: [http://huai:35297/]
[xmlrpc][INFO] 2020-01-28 23:05:07,936: Started XML-RPC server [http://huai:35297/]
[rospy.impl.masterslave][INFO] 2020-01-28 23:05:07,936: _ready: http://huai:35297/
[rospy.registration][INFO] 2020-01-28 23:05:07,941: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-28 23:05:07,941: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-28 23:05:08,036: registered with master
[rospy.rosout][INFO] 2020-01-28 23:05:08,036: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-28 23:05:08,038: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-28 23:05:08,039: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-28 23:05:08,243: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-28 23:05:10,446: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-28 23:05:36,426: topic[/cam0/image_raw] adding connection to [http://huai:35829/], count 0
[rospy.core][INFO] 2020-01-28 23:05:49,104: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-28 23:05:49,106: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-28 23:05:49,107: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-01-28 23:05:49,107: topic[/cam0/image_raw] removing connection to http://huai:35829/
[rospy.internal][WARNING] 2020-01-28 23:05:49,107: Unknown error initiating TCP/IP socket to huai:50059 (http://huai:35829/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-01-28 23:05:49,113: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 841, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-01-28 23:05:49,113: signal-2
[rospy.core][INFO] 2020-01-28 23:05:49,259: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-29 00:18:06,191: init_node, name[/ros_test], pid[16910]
[xmlrpc][INFO] 2020-01-29 00:18:06,197: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-29 00:18:06,198: Started XML-RPC server [http://huai:34697/]
[rospy.init][INFO] 2020-01-29 00:18:06,198: ROS Slave URI: [http://huai:34697/]
[rospy.impl.masterslave][INFO] 2020-01-29 00:18:06,198: _ready: http://huai:34697/
[rospy.registration][INFO] 2020-01-29 00:18:06,203: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-29 00:18:06,203: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-29 00:18:06,298: registered with master
[rospy.rosout][INFO] 2020-01-29 00:18:06,299: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-29 00:18:06,300: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-29 00:18:06,301: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-29 00:18:06,539: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-01-29 00:18:08,370: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-29 00:18:08,371: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2020-01-29 00:18:08,371: signal-2
[rospy.core][INFO] 2020-01-29 00:18:08,581: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-29 00:18:23,208: init_node, name[/ros_test], pid[17175]
[xmlrpc][INFO] 2020-01-29 00:18:23,213: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-29 00:18:23,213: Started XML-RPC server [http://huai:35271/]
[rospy.init][INFO] 2020-01-29 00:18:23,213: ROS Slave URI: [http://huai:35271/]
[rospy.impl.masterslave][INFO] 2020-01-29 00:18:23,214: _ready: http://huai:35271/
[rospy.registration][INFO] 2020-01-29 00:18:23,219: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-29 00:18:23,219: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-29 00:18:23,314: registered with master
[rospy.rosout][INFO] 2020-01-29 00:18:23,314: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-29 00:18:23,316: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-29 00:18:23,317: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-29 00:18:23,565: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-01-29 00:18:25,433: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-29 00:18:25,435: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2020-01-29 00:18:25,435: signal-2
[rospy.core][INFO] 2020-01-29 00:18:26,095: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-29 00:18:42,501: init_node, name[/ros_test], pid[17469]
[xmlrpc][INFO] 2020-01-29 00:18:42,509: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-29 00:18:42,510: Started XML-RPC server [http://huai:45903/]
[rospy.impl.masterslave][INFO] 2020-01-29 00:18:42,510: _ready: http://huai:45903/
[rospy.init][INFO] 2020-01-29 00:18:42,511: ROS Slave URI: [http://huai:45903/]
[rospy.registration][INFO] 2020-01-29 00:18:42,519: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-29 00:18:42,519: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-29 00:18:42,611: registered with master
[rospy.rosout][INFO] 2020-01-29 00:18:42,612: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-29 00:18:42,613: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-29 00:18:42,614: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-29 00:18:42,894: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-29 00:18:44,997: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-29 00:19:41,273: topic[/cam0/image_raw] adding connection to [http://huai:38667/], count 0
[rospy.internal][WARNING] 2020-01-29 00:22:08,767: Unknown error initiating TCP/IP socket to huai:35915 (http://huai:38667/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-29 00:22:08,767: topic[/cam0/image_raw] removing connection to http://huai:38667/
[rospy.core][INFO] 2020-01-29 18:28:26,382: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-29 18:28:26,384: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-29 18:28:26,384: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-29 18:28:26,384: signal-2
[rospy.core][INFO] 2020-01-29 18:28:26,549: signal_shutdown [atexit]
[rospy.client][INFO] 2020-01-29 18:28:32,019: init_node, name[/ros_test], pid[17659]
[xmlrpc][INFO] 2020-01-29 18:28:32,020: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-29 18:28:32,020: Started XML-RPC server [http://huai:36147/]
[rospy.impl.masterslave][INFO] 2020-01-29 18:28:32,021: _ready: http://huai:36147/
[rospy.init][INFO] 2020-01-29 18:28:32,021: ROS Slave URI: [http://huai:36147/]
[rospy.registration][INFO] 2020-01-29 18:28:32,021: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-29 18:28:32,021: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-29 18:28:32,122: registered with master
[rospy.rosout][INFO] 2020-01-29 18:28:32,122: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-29 18:28:32,124: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-29 18:28:32,126: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-29 18:28:32,367: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-01-29 18:28:34,965: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-01-29 18:28:54,405: topic[/cam0/image_raw] adding connection to [http://huai:46621/], count 0
[rospy.internal][WARNING] 2020-01-29 18:31:22,009: Unknown error initiating TCP/IP socket to huai:33161 (http://huai:46621/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-01-29 18:31:22,009: topic[/cam0/image_raw] removing connection to http://huai:46621/
[rospy.core][INFO] 2020-01-29 19:04:48,884: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-29 19:04:48,892: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-29 19:04:48,893: topic[/Lines2d] removing connection to /tracking_node
[rospy.impl.masterslave][INFO] 2020-01-29 19:04:48,893: signal-2
[rospy.core][INFO] 2020-01-29 19:04:49,025: signal_shutdown [atexit]
